core/domain/rating_services.py

```diff
@@ -57,6 +57,9 @@ def assign_rating_to_exploration(user_id, exploration_id, new_rating):
         raise Exception('Invalid exploration id %s' % exploration_id)
 
     def _update_user_rating():
+        """Updates the user rating of the exploration. Returns the old rating
```

Comment:
I think a single sentence would be better without a break, what do you say?  `and returns the old rating....`

Reply:
Hi @bansalnitish, I did exactly the same as in your both the comments. But after @nithusha21's review, I made these changes. I think both the ways are acceptable but it's just the matter of individual preferences.

Reply:
Okay, sounds good!

Reply:
Typically, we would have a "returns" part of the docstring, I wanted to have it somewhat resemble that. That's why the split sentences. But yeah, I think this does come down to personal preference :P